content/en/user-guide/aws/codepipeline/index.md

```diff
@@ -0,0 +1,446 @@
+                },
+                "region": "eu-central-1",
+                "inputArtifacts": [
+                    {
+                        "name": "intermediate-file",
+                        "s3location": {
+                            "bucket": "artifact-store-bucket",
+                            "key": "pipeline/intermediate-file/01410aa4.zip"
+                        }
+                    }
+                ]
+            },
+            "output": {
+                "outputArtifacts": [],
+                "executionResult": {
+                    "externalExecutionId": "bcff0781",
+                    "externalExecutionSummary": "Deployment Succeeded"
+                },
+                "outputVariables": {}
+            }
+        },
+        {
+            "pipelineExecutionId": "37e8eb2e-0ed9-447a-a016-8dbbd796bfe7",
+            "actionExecutionId": "ae99095a-1d43-46ee-8a48-c72b6d60021e",
+            "pipelineVersion": 1,
+            "stageName": "stage1",
+            "actionName": "action1",
+            ...
+</disable-copy>
+{{< /command >}}
+
+{{< callout >}}
+LocalStack does not use the same logic to generate external execution IDs as AWS so there may be minor discrepancies.
+Same is true for status and error messages produced by actions.
+{{< /callout >}}
+
+## Pipelines
+
+The operations [CreatePipeline](https://docs.aws.amazon.com/codepipeline/latest/APIReference/API_CreatePipeline.html), [GetPipeline](https://docs.aws.amazon.com/codepipeline/latest/APIReference/API_GetPipeline.html), [UpdatePipeline](https://docs.aws.amazon.com/codepipeline/latest/APIReference/API_UpdatePipeline.html), [ListPipelines](https://docs.aws.amazon.com/codepipeline/latest/APIReference/API_ListPipelines.html), [DeletePipeline](https://docs.aws.amazon.com/codepipeline/latest/APIReference/API_DeletePipeline.html) are used to manage pipeline declarations.
+
+LocalStack supports emulation for V1 pipelines.
+V2 pipelines are only created as mocks.
+
+{{< callout "tip" >}}
+Emulation for V2 pipelines is not supported.
+Make sure that the pipeline type is explicitly set in the declaration.
+{{< /callout >}}
+
+Pipeline executions can be managed with [StartPipelineExecution](https://docs.aws.amazon.com/codepipeline/latest/APIReference/API_StartPipelineExecution.html), [GetPipelineExecution](https://docs.aws.amazon.com/codepipeline/latest/APIReference/API_GetPipelineExecution.html), [ListPipelineExecutions](https://docs.aws.amazon.com/codepipeline/latest/APIReference/API_ListPipelineExecutions.html) and [StopPipelineExecutions](https://docs.aws.amazon.com/codepipeline/latest/APIReference/API_StopPipelineExecution.html).
+
+When stopping pipeline executions with StopPipelineExecution, the stop and abandon method is not supported.
+Setting the `abandon` flag will have no impact.
+This is because LocalStack uses threads as the underlying mechanism to simulate pipelines, and threads can not be cleanly preempted.
```

Comment:
```suggestion
This is because LocalStack uses threads as the underlying mechanism to simulate pipelines, and threads cannot be cleanly preempted.   
```